data/language/en-GB.txt

```diff
@@ -3673,6 +3673,7 @@ STR_6356    :{SMALLFONT}{BLACK}Spawns ducks if park contains water
 STR_6357    :{SMALLFONT}{BLACK}Removes all ducks from the map
 STR_6358    :Page {UINT16}
 STR_6359    :{POP16}{POP16}Page {UINT16}
+STR_6360    :{SMALLFONT}{BLACK}{UINT16}
```

Comment:
What's this being used for?

Reply:
This string is used to display the y-axis labels.

Come to think of it, perhaps I should have used `{COMMA16}` instead, so we'd also get separators; e.g. `1,000` instead of just `1000`.

Reply:
For the guest numbers, you will need to use COMMA32 - the new save format will allow more than the 32767 guests that a COMMA16 can represent.

Reply:
I don't mind the change, but at that point, the scales will require changing, anyway…

Reply:
At least in that case, it can't overflow...

Reply:
Yes, but since we know this is coming up and we're touching this code anyway, I would argue we should future-proof it now. The new save format PR is big and far-reaching enough as it is.